source/globalCommands.py

```diff
@@ -901,6 +925,20 @@ def script_toggleReportLinks(self, gesture):
 			config.conf["documentFormatting"]["reportLinks"] = True
 		ui.message(state)
 
+	@script(
+		# Translators: Input help mode message for toggle report link type command.
+		description=_("Toggles on and off the reporting of link type"),
+		category=SCRCAT_DOCUMENTFORMATTING,
+	)
+	def script_toggleReportLinkType(self, gesture: inputCore.InputGesture):
+		msg = toggleBooleanValue(
+			configSection="documentFormatting",
+			configKey="reportLinkType",
+			enabledMsg="Report link type on",
+			disabledMsg="Report link type off",
```

Comment:
these need translation comments

```suggestion
			enabledMsg=_("Report link type on"),
			disabledMsg=_("Report link type off"),
```